filesystem/src/test/java/org/frankframework/filesystem/SftpFileSystemTest.java

```diff
@@ -46,28 +41,14 @@ private void startNewSshDaemon() throws IOException {
 		if("localhost".equals(host)) {
 			remoteDirectory = "/"; // See getTestDirectoryFS(), '/' is the SFTP HOME directory.
 
-			sshd = createSshServer(username, password);
+			sshd = SftpFileSystemTestHelper.createSshServer(username, password, port);
+			sshd.setFileSystemFactory(getTestDirectoryFS());
```

Comment:
Deze zit nu in de helper toch?

Reply:
Ja oeps, die had ik nog willen opruimen. Done. Thanks.